app/Http/Controllers/CategoriesController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Http\Requests\ValidateCategory;
+use App\Models\Categories;
+use Illuminate\Http\Request;
+use Illuminate\Support\Str;
+
+class CategoriesController extends Controller
+{
+    public function index()
+    {
+        $categories = Categories::all();
+
+        return view('sections.categories.categories-page', ['categories' => $categories]);
+    }
+
+    public function createCategory()
```

Comment:
Mantém a consistência de nomenclatura dos metodos. No caso dessa controller todos os outros metodos não mencionam categoria no nome.
```suggestion
    public function create()
```